packages/cxl-ui/scss/cxl-dashboard-header.scss

```diff
@@ -50,6 +50,22 @@
       }
     }
 
+    .subscription-status {
+      font-family: var(--cxl-lumo-font-ternary);
+      font-size: var(--lumo-font-size-s);
+      margin-bottom: var(--lumo-space-s);
+
+      a {
+        color: var(--lumo-primary-color);
+        margin-left: var(--lumo-space-s);
+        text-decoration: none;
+
+        &:hover {
+          text-decoration: underline;
+        }
```

Comment:
@anoblet @lkraav why do I need to define styles for links inside the component while they're [defined globally](https://conversionxl.github.io/aybolit/?path=/story/cxl-lumo-styles-typography--base)? Would I need to include some global scss? Is it a good idea?

Reply:
Are these links in Shadow DOM?

Reply:
I noticed this as well: https://cxlworld.slack.com/archives/C01JABH8AHX/p1704297049011649

You should be able to `@use`, `@import` the typography library: https://github.com/conversionxl/aybolit/blob/c9114aa6564034923007a9b77bd74bf93befdd69/packages/cxl-ui/scss/cxl-certificate-header.scss#L3

Reply:
I just noticed this style is already being imported. It doesn't have `a` styles though